controllers/users/users.js

```diff
@@ -40,58 +42,80 @@ exports.activateUser = (req, res) => {
     .catch(() => {
       res.status(400).json({ msg: messages.SERVER_ERROR });
     });
-}
+};
 
 exports.registerUser = (req, res) => {
   const errors = validationResult(req);
 
   if (!errors.isEmpty()) {
-    return res.status(400).json({ errors: errors.array({ onlyFirstError: true }) });
+    return res
+      .status(400)
+      .json({ errors: errors.array({ onlyFirstError: true }) });
   }
 
-  const { firstName, lastName, email = email.toLowerCase(), password } = req.body;
+  const {
+    firstName,
+    lastName,
+    email = email.toLowerCase(),
+    password
+  } = req.body;
 
   User.findOne({ email }).then(user => {
     if (user) {
-      return res.status(400).json({ errors: [{ msg: messages.EMAIL_ALREADY_EXISTS }] });
+      return res
+        .status(400)
+        .json({ errors: [{ msg: messages.EMAIL_ALREADY_EXISTS }] });
     } else {
-      const newUser = new User({ 
-        firstName, 
-        lastName, 
-        email, 
-        password: bcrypt.hashSync(password), 
-        activationKey: bcrypt.hashSync(email).replace(/\//g, '') 
+      const newUser = new User({
+        firstName,
+        lastName,
+        email,
+        password: bcrypt.hashSync(password),
+        activationKey: bcrypt.hashSync(email).replace(/\//g, "")
       });
 
-      newUser.save()
+      newUser
+        .save()
         .then(user => {
-          const payload = { 
-            user: { 
-              id: user.id 
-            } 
+          const payload = {
+            user: {
```

Comment:
User id can be stored in subject field

{
 sub: user.id
}

Maybe passport js provide us with some support for creation of JWT as well